product_pricelist_item_button/models/product_pricelist_item.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import _, models
+
+
+class ProductPricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    def show_rules(self):
```

Comment:
Maybe show_items could be a better name

Reply:
I've followed Odoo display names :-)

Reply:
But I can change

Reply:
matter of taste. i usually use technical name for technical name function. but really not a blocking point !